### PR TITLE
[LogitProcessor] Use min float value as the mask value

### PR DESCRIPTION
**Pull request by MasterJH5574**
This PR updates the mask values in LogitProcessor to the min value of float32. Prior to this PR it was -1e10. This update is the safest for softmax as long as the masking is always the last step in logit processor.

### CODE REVIEW

**Review by tqchen — APPROVED** [review]

**Comment by jpf888** [comment]
@MasterJH5574

i suspect that there might be an issue with how the repetition_penalty is being handled.
i believe the logic should be ：
if logits[seq_ids[pos2seq_id[vp]], token_ids[vp]] > 0, logits[seq_ids[pos2seq_id[vp]], token_ids[vp]] / penalties[pos2seq_id[vp], 2].

The origin code here ：
        logits[seq_ids[pos2seq_id[vp]], token_ids[vp]] = T.if_then_else( logits[seq_ids[pos2seq_id[vp]], token_ids[vp]] > 0, logits[seq_ids[pos2seq_id[vp]], token_ids[vp]] * penalties[pos2seq_id[vp], 2], logits[seq_ids[pos2seq_id[vp]], token_ids[vp]] / penalties[pos2seq_id[vp], 2],

might be reversed？？

**Comment by MasterJH5574** [comment]
@jpf888 thanks for reporting this. It is fixed in https://github.com/mlc-ai/mlc-llm/pull/2937.
